--- libcroco-0.6.1-orig/configure.in	2006-03-05 16:13:51.000000000 -0600
+++ libcroco-0.6.1/configure.in	2006-03-07 18:46:10.997625000 -0600
@@ -58,14 +58,14 @@
 AM_PROG_LIBTOOL
 
 case "$host" in
-  *-*-mingw*)
-    os_win32=yes
+  *-*-cygwin*|*-*-mingw*)
+    platform_win32=yes
     ;;
   *)
-    os_win32=no
+    platform_win32=no
     ;;
 esac
-AM_CONDITIONAL(OS_WIN32, [test "$os_win32" = "yes"])
+AM_CONDITIONAL(PLATFORM_WIN32, [test "$platform_win32" = "yes"])
 
 GTK_DOC_CHECK([1.0])
 
--- libcroco-0.6.1-orig/src/Makefile.am	2006-03-05 15:57:02.000000000 -0600
+++ libcroco-0.6.1/src/Makefile.am	2006-03-07 18:43:49.685125000 -0600
@@ -1,4 +1,4 @@
-if OS_WIN32
+if PLATFORM_WIN32
 no_undefined = -no-undefined
 endif
 
